File: wire-android-automation/common/src/main/java/com/wearezeta/auto/common/email/inbucket/models/Attachment.java
package com.wearezeta.auto.common.email.inbucket.models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public class Attachment {
    private String filename;
    @JsonProperty("content-type")
    private String contentType;
    @JsonProperty("download-link")
    private String downloadLink;
    @JsonProperty("view-link")
    private String viewLink;
    private String md5;

}

File: tests/testsCore/src/main/kotlin/com/wire/android/tests/core/models/Attachment.kt
/*
 * Wire
 * Copyright (C) 2025 Wire Swiss GmbH
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program. If not, see http://www.gnu.org/licenses/.
 */
package com.wire.android.tests.core.models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
class Attachment {
    private val filename: String? = null

    @JsonProperty("content-type")
    private val contentType: String? = null

    @JsonProperty("download-link")
    private val downloadLink: String? = null

    @JsonProperty("view-link")
    private val viewLink: String? = null
    private val md5: String? = null
}
